### ISSUE TITLE
Link to website gets a 404

### ISSUE THREAD

**Issue by Gaurav0** [rwjblue/heimdalljs-visualizer#18]

**Comment by rwjblue** [comment]
Thank you, I’ll get it working again. 

FWIW - the issue is that I transferred the project into the org, and the gh-pages setup doesn’t do redirects :(

**Comment by patocallaghan** [comment]
Yep discovered this myself just now. New URL is https://heimdalljs.github.io/heimdalljs-visualizer/

**Comment by rwjblue** [comment]
Updated the URL in the repo description, but I still want to deploy something (preferably a redirect) at the old location just to avoid trolling folks.
